playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_max_width.html.erb

```diff
@@ -17,4 +17,8 @@
     <%= pb_rails("title", props: { text: "SM", size: 4, classname: "dark" }) %>
```

Comment:
You need to add spacing between the last one. 

![image](https://user-images.githubusercontent.com/863031/143905189-a76cd054-5c10-4588-b6dc-9b86eb2e8c85.png)